fstar-helpers/minicore/src/arch/x86.rs

```diff
@@ -0,0 +1,289 @@
+/// This is a (partial) mirror of [`core::arch::x86`] and [`core::arch::x86_64`].
```

Comment:
This is not a mirror. The functions have completely different function signatures. Please let us know what it is supposed to be.

Reply:
This crate is supposed to be a partial model of libcore, meaning that every item that exists both in core and in minicore should share the same type signature.

I think that mostly holds. I would like a CI to check that, though.

That's what I mean by mirror, I will update the comment to explain that.

Reply:
I pushed e33ea12b